Source/Private/Convert-ToDockerHostPath.ps1

```diff
@@ -0,0 +1,24 @@
+function Convert-ToDockerHostPath {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [String]
+        $Path
+    )
+
+    $pathOnDockerHost = $Path
+    if ( (Invoke-DockerCommand 'ps').StdOut | Select-String $(hostname) ) { 
+        # executed inside docker container $(hostname)
+        $dockerCommand = "inspect -f ""{{ range .Mounts }}{{ .Source }}:{{ .Destination }}{{ println }} {{ end }}"" $(hostname)"
+        $mounts = (Invoke-DockerCommand $dockerCommand ).StdOut.trim() | Where-Object {  $_ -NotMatch "/var/lib/docker" -and $_ -NotMatch "docker.sock" -and $_ -ne '' }
```

Comment:
This command outputs some logs too. It should be silenced.

Reply:
Output is redirected to $mounts. No noise in log is expected.

Reply:
Sample logs:

```
09:30:00  CONTAINER ID        IMAGE                                                                    COMMAND             CREATED             STATUS              PORTS               NAMES
09:30:00  725ff90cb743        jenkins.agent.linux:latest   "jenkins-agent"     41 minutes ago      Up 41 minutes                           jenkins.agent
09:30:01  /home/devops/workspace:/home/jenkins/workspace
09:30:01   /var/run/docker.sock:/var/run/docker.sock
09:30:01   /var/lib/docker/volumes/8074c6c7f252fb81b252e21895bcb7dcc5f1b8433077db224b14c74604032720/_data:/home/jenkins/.jenkins
09:30:01   /var/lib/docker/volumes/b722acbfc500dbb30b7a62a01153e880ff4541a1481e5391be06820cffedf598/_data:/home/jenkins/agent
09:30:01   
```
Also applicable to the above comment.

Reply:
fixed with -Quiet switch